### DISCUSSION TITLE
debian11安装面板失败

### DISCUSSION THREAD

**Discussion by MingLoong** [midoks/mdserver-web#472] [Q&A] [answered]
[mw-install.log](https://github.com/midoks/mdserver-web/files/12844609/mw-install.log)
已上传安装日志

**Comment by 0honus0** [comment, marked as answer]
看代码是 https://github.com/midoks/mdserver-web/blob/780dfb08a30af4996ef53fdb559befe8d24672c1/scripts/lib.sh#L115 这里的问题
1. 可以手动在 https://github.com/midoks/mdserver-web/tree/master/version 下面创建对应版本的 r3.xx.txt 文件，xx对应python小版本，里面写入 flask==2.3.3
2. 可以修改 https://github.com/midoks/mdserver-web/blob/780dfb08a30af4996ef53fdb559befe8d24672c1/requirements.txt#L7  flask==2.3.3

1或者2任选一种，然后手动执行 bash /www/server/mdserver-web/scripts/lib.sh，执行完成重启即可

**Comment by midoks** [comment]
已经合并，mw update，即可正常使用
